pyserver/main.py

```diff
@@ -312,6 +319,7 @@ def all_comments_to_claims(req:CommentTopicTree, log_to_wandb:bool = False) -> d
                     "claims": [
                         {
                             "claim": "Cats are the best household pets.",
+                            commentId":"..",
```

Comment:
same minor nit: commentId missing starting quote, would prefer literal example of a comment id string